test/blackbox-tests/test-cases/actions/conc.t

```diff
@@ -0,0 +1,107 @@
+Specification of the concurrency action:
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.6)
+  > EOF
+
+  $ cat > dune << EOF
+  > (rule
+  >  (action
+  >   (conc )))
+  > EOF
+
+  $ dune build
+  File "dune", line 3, characters 2-9:
+  3 |   (conc )))
+        ^^^^^^^
+  Error: 'conc' is only available since version 3.7 of the dune language.
+  Please update your dune-project file to have (lang dune 3.7).
```

Comment:
```suggestion
  Error: 'conc' is only available since version 3.7 of the Dune language.
  Please update your `dune-project` file to have `(lang dune 3.7)`.
```